src/Okta.Sdk.IntegrationTests/LogsScenarios.cs

```diff
@@ -103,6 +103,13 @@ public async Task GetLogsByEventType()
         [Trait("Category", "NoBacon")] // Tests that don't run on internal CI pipeline
         public async Task GetLogsBySinceDate()
         {
+            // Don't run test on travis - https://github.com/okta/okta-sdk-dotnet/issues/271
+            string travis = System.Environment.GetEnvironmentVariable("TRAVIS");
```

Comment:
Let's `[Skip]` tests that are currently broken rather than hard code env var checks.

Reply:
Done